Content.Server/Chemistry/Metabolism/MovespeedModifierMetabolism.cs

```diff
@@ -0,0 +1,63 @@
+using Content.Shared.Chemistry.Reagent;
+using Robust.Shared.GameObjects;
+using Robust.Shared.Serialization.Manager.Attributes;
+using Content.Server.GameObjects.Components.Chemistry;
+using System.Threading;
+using Content.Shared.Chemistry.Metabolizable;
+using Content.Shared.Movement.Components;
+
+namespace Content.Server.Chemistry.Metabolism
+{
+    /// <summary>
+    /// Default metabolism for stimulants and tranqs. Attempts to find a MovementSpeedModifier on the target,
+    /// adding one if not there and to change the movespeed
+    /// </summary>
+    [DataDefinition]
+    public class MovespeedModifierMetabolism : IMetabolizable
+    {
+
+        /// <summary>
+        /// How much of the reagent should be metabolized each sec.
+        /// </summary> 
+        [DataField("rate")]
+        public ReagentUnit MetabolismRate { get; set; } = ReagentUnit.New(1);
+
+        /// <summary>
+        /// Walk speed modifier
```

Comment:
these doccomments should either explain further or just be removed entirely, if a comment just repeats the name of the variable it's not needed

Reply:
also uhhhhhh 10 seems very high?